src/status_im/ui/screens/rpc_usage_info.cljs

```diff
@@ -45,6 +46,12 @@
      :params []
      :on-success #(log/debug "rpcstats_reset success")})))
 
+(defonce rpc-refresh-interval (atom nil))
+
+(def rpc-usage-refresh-interval 2000)
+
+(reset! rpc-refresh-interval (utils/set-interval #(re-frame/dispatch [::get-stats]) rpc-usage-refresh-interval))
```

Comment:
i guess we need this only when the screen is opened

Reply:
Yes, probably best to use the component lifecycle methods (`componentdidmount` & `componentwillunmount` for cleaning up), there should be a few examples on how we do this in the codebase (if you search for `component-did-mount` you should find a few examples)

Reply:
@flexsurfer @cammellos - Updated the code with interval cleanup

Reply:
@smohamedjavid could you run `make lint-fix` and `make lint` please? The build is failing because of indentation.
Thanks!

Reply:
@cammellos - Apologies, I have fixed the lint issue and the build is successful now.